src/Domain/Strava/Ftp/EFtpRepository.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Domain\Strava\Ftp;
+
+use App\Domain\Strava\Activity\Activities;
+use App\Domain\Strava\Activity\ActivityType;
+use App\Infrastructure\ValueObject\Time\SerializableDateTime;
+
+interface EFtpRepository
```

Comment:
Repositories should only be responsible for fetching data, in this case querying the sqlite db. They should not be able to:
* getNumberOfMonths()
* enabled()
* enrichWithActivities(Activities $allActivities)

This repository has state which we definitely don't want